pages/docs/integrations/llama-index/get-started.mdx

```diff
@@ -3,7 +3,11 @@ title: Open Source Observability for LlamaIndex
 description: Open source observability for LlamaIndex. Automatically capture detailed traces and metrics for every request of your RAG application.
 ---
 
-# 🦙 LlamaIndex Integration
+# 🦙 LlamaIndex Integration (SDK v2)
+
+<Callout type="info">
```

Comment:
need to update page that uses the v3 sdk + instrumentation, this is not needed anymore

until then, pin v2 sdk in installs, otherwise the code examples here dont work